src/main/fc/cli.c

```diff
@@ -2081,29 +2081,38 @@ static void printMap(uint8_t dumpMask, const rxConfig_t *rxConfig, const rxConfi
 
 static void cliMap(char *cmdline)
 {
-    uint32_t len;
-    char out[9];
+    uint32_t i;
+    char buf[MAX_MAPPABLE_RX_INPUTS + 1];
 
-    len = strlen(cmdline);
+    uint32_t len = strlen(cmdline);
+    if (len == MAX_MAPPABLE_RX_INPUTS) {
 
-    if (len == 8) {
-        // uppercase it
-        for (uint32_t i = 0; i < 8; i++)
-            cmdline[i] = toupper((unsigned char)cmdline[i]);
-        for (uint32_t i = 0; i < 8; i++) {
-            if (strchr(rcChannelLetters, cmdline[i]) && !strchr(cmdline + i + 1, cmdline[i]))
+        for (i = 0; i < MAX_MAPPABLE_RX_INPUTS; i++) {
+            buf[i] = toupper((unsigned char)cmdline[i]);
+        }
+        buf[i] = '\0';
+
+        for (i = 0; i < MAX_MAPPABLE_RX_INPUTS; i++) {
+            buf[i] = toupper((unsigned char)cmdline[i]);
```

Comment:
Oh wait I thought you were referring to something else. Heh,  yes the second toupper line should have been deleted but the upper stuff *Must* be done first.